tdrs-frontend/src/components/Welcome/Welcome.scss

```diff
@@ -0,0 +1,16 @@
+.welcome {
+  text-align: center;
+  padding-top: 30vh;
+  .left {
+    padding: 19px 50px 0 0;
```

Comment:
I know that this will get eliminated by implementing the mockups, but I'd prefer if you use the `uswds` spacing token mix-ins where possible: https://designsystem.digital.gov/design-tokens/spacing-units/